src/main/java/com/epam/ms/config/RabbitConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.ms.config;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.core.AmqpAdmin;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.rabbit.connection.CachingConnectionFactory;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.core.RabbitAdmin;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Slf4j
+@Configuration
+public class RabbitConfiguration {
+    private static final String HOST = "localhost";
```

Comment:
It's a best practice to have all the configurations to be defined in `application.yml` file.
Then you use `@Value("${my.custom.property.path}") String customPath;` to access this configuration

Reply:
Thank you, done